.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,44 @@
+name: Production deployment
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  id-token: write   
+  contents: read
+jobs:
+  deploy-kvittering-backend:
+    name: Deploy monoweb/kvittering-backend
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-north-1
+          role-to-assume: arn:aws:iam::891459268445:role/MonowebProdKvitteringBackendCIRole
+      - uses: aws-actions/amazon-ecr-login@v2
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      - uses: docker/build-push-action@v6
+        with:
+          platforms: linux/amd64
+          context: .
+          file: apps/kvittering-backend/Dockerfile
+          tags: 891459268445.dkr.ecr.eu-north-1.amazonaws.com/monoweb/kvittering-backend:latest
```

Comment:
These should be per-environment, so monoweb/prod/kvittering-backend, you will probably need infra changes here too

Reply:
True, nice catch